### PR TITLE
Fix API GW tests

### PR DESCRIPTION
**Pull request by nitinprabhu**

### CODE REVIEW

File: src/smokeTest/resources/test-certificates.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Only valid certificates are stored in key vault.
+As inspected in test suite value is base64 encoded.
+Here is the command line which achieved desired outcome:
+
+```bash
+openssl base64 -in <infile> -out <outfile>
+```
+
+- `<infile>` would be certificate, e.g. `valid-certificate.jks`
```
Comment (doncem) [inline src/smokeTest/resources/test-certificates.md]:
```suggestion
- `<infile>` would be certificate, e.g. `valid-certificate.pfx`
```

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -14,27 +14,38 @@ def component = "blob-router"
...
+
+     secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
+
+     secret('storage-account-secondary-key', 'TEST_STORAGE_ACCOUNT_KEY'),
+     secret('storage-account-name', 'TEST_STORAGE_ACCOUNT_NAME'),
+
+     secret('crime-storage-connection-string', 'STORAGE_CRIME_CONNECTION_STRING'),
+     secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
+     secret('test-valid-key-store', 'TEST_VALID_KEY_STORE'),
+     secret('test-valid-key-store-password', 'TEST_VALID_KEY_STORE_PASSWORD')
```
Comment (doncem) [inline Jenkinsfile_CNP]: those are already defined

**Comment by lgonczar** [comment]
We need two more tests - for not yet valid certificate and for an expired one. That can be done later, as they don't exist in the processor, either.

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -13,28 +13,36 @@ def component = "blob-router"
...
+
+     secret('crime-storage-connection-string', 'STORAGE_CRIME_CONNECTION_STRING'),
+     secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
+     secret('test-valid-key-store', 'TEST_VALID_KEY_STORE'),
+     secret('test-valid-key-store-password', 'TEST_VALID_KEY_STORE_PASSWORD'),
+
+     // smoke test secrets
+     secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
+     secret('test-valid-key-store', 'TEST_VALID_KEY_STORE'),
+     secret('test-valid-key-store-password', 'TEST_VALID_KEY_STORE_PASSWORD')
```
Comment (nitinprabhu) [inline Jenkinsfile_CNP]: duplicates ?

Comment (lgonczar) [reply Jenkinsfile_CNP]: It's either one set or another, from what I can see

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -13,28 +13,28 @@ def component = "blob-router"
...
-    secret('crime-storage-connection-string', 'STORAGE_CRIME_CONNECTION_STRING')
-  ]
-]
+ def smokeTestSecrets = [
+   'reform-scan-${env}': [
+     secret('test-subscription-key', 'TEST_SUBSCRIPTION_KEY'),
+     secret('test-valid-key-store', 'TEST_VALID_KEY_STORE'),
+     secret('test-valid-key-store-password', 'TEST_VALID_KEY_STORE_PASSWORD')
+   ]
+ ]
```
Comment (doncem) [inline Jenkinsfile_CNP]: ah so that's what changed the block - extra space

**Pull request merged**